util/from-random/README.md

```diff
@@ -0,0 +1,24 @@
+A trait for constructing an object from a random number generator.
+
+This provides a common API for types which can be initialized from random number generators.
+
+# Example
+
+```rust
+use mc_from_random::FromRandom;
+use rand_core::{CryptoRng, RngCore};
+
+struct MyStruct {
+    pub bytes: [u8; 32],
+}
+
+impl FromRandom for MyStruct {
+    fn from_random<R: CryptoRng, RngCore>(csprng: &mut R) -> Self {
+        let mut bytes = [0u8; 32];
+        csprng.fill_bytes(&bytes);
+        MyStruct {
+            bytes,
+        }
+    }
+}
+```
```

Comment:
this is nice, I guess I should put more actual example code in my readme's or something

Reply:
Yeah this is great - speaking of, @jcape you could throw in the `#![doc(include = "../README.md")]` at the top of lib.rs.